onmt/model_builder.py

```diff
@@ -139,9 +139,9 @@ def load_test_model(opt, model_path=None):
 
     model_opt = ArgumentParser.ckpt_model_opts(checkpoint['opt'])
     # Patch for NLLB200 model loading
-    if ('encoder.embeddings.make_embedding.pe.pe' not in
-            checkpoint['model'].keys()):
-        model_opt.position_encoding_type = 'SinusoidalConcat'
+    # if ('encoder.embeddings.make_embedding.pe.pe' not in
```

Comment:
remove if obsolete

Reply:
I know but I need to update the NLLB-200 checkpoints first so that this is no longer needed. Just a reminder.